### PR TITLE
#160141033 Display correct meeting room details

### PR DESCRIPTION
**Pull request by Rwothoromo**
#### What does this PR do?
Adds functionality to display the correct meeting room data.

#### Description of Task to be completed?
- On clicking a meeting room button, show the actual room details.

#### How should this be manually tested?
- Run the application.
- Select a meeting room from a list of rooms; the room name is displayed correctly.
- Go ahead and select `Room info`; the correct details are shown.

#### Any background context you want to provide?
Prior to this PR, the default meeting room name in the layout file is what was being displayed.

#### What are the relevant pivotal tracker stories?
[#160141033](https://www.pivotaltracker.com/story/show/160141033)

#### Screenshots (if appropriate)
ezgif com-video-to-gif 2

### CODE REVIEW

File: app/src/main/java/com/andela/mrm/room_setup/country/CountryActivity.java [hunk 1]
```diff
@@ -23,23 +35,20 @@
...
-    public static final String SHARED_PREF_FILENAME = "room_setup";
-    public static final String SHARED_COUNTRY_ID = "country_id";
-    public static final String SHARED_OFFICE_ID = "office_id";
-    public static final String SHARED_BUILDING_ID = "building_id";
-    public static final String SHARED_FLOOR_ID = "floor_id";
-    public static final String SHARED_ROOM_ID = "room_id";
-    public static final String SHARED_ROOM_ALREADY_SETUP_KEY = "room_already_setup";
-
+    public static Intent newIntent(Context context) {
+        return new Intent(context, CountryActivity.class);
```
Comment (TheDancerCodes) [inline app/src/main/java/com/andela/mrm/room_setup/country/CountryActivity.java]: Hi @Rwothoromo 
There seems to be a drop in total code coverage as result of this line. 

Could you please write some test(s) for this functionality. 
Code Climate Plugin Screenshot

Comment (Rwothoromo) [reply app/src/main/java/com/andela/mrm/room_setup/country/CountryActivity.java]: Hi @TheDancerCodes 
Does the coverage change still hold?

**Comment by TheDancerCodes** [comment]
@Rwothoromo The difference has improved from `-1.8%` to `-0.6%` as evidenced in the screenshot.

Code Climate Screenshot

You can monitor this from your end by installing the [Code Climate Chrome Extension](https://chrome.google.com/webstore/detail/code-climate/phgahogocbnfilkegjdpohgkkjgahjgk?hl=en) and viewing this directly from Github.

File: app/src/main/java/com/andela/mrm/room_setup/country/CountryActivity.java (hunk 1 above)
Comment (TheDancerCodes) [reply app/src/main/java/com/andela/mrm/room_setup/country/CountryActivity.java]: @Rwothoromo The difference has improved from `-1.8%` to `-0.6%`.

Comment (Rwothoromo) [reply app/src/main/java/com/andela/mrm/room_setup/country/CountryActivity.java]: @TheDancerCodes The coverage is finally fixed

**Comment by Rwothoromo** [comment]
Thank you @murageden

**Comment by TheDancerCodes** [comment]
@Rwothoromo 
Kindly pull the changes in develop and rebase before I merge this.

**Comment by Rwothoromo** [comment]
@TheDancerCodes I'll do that right away.

File: app/src/androidTestMock/java/com/andela/mrm/room_availability/RoomAvailabilityActivityTest.java [hunk 2]
```diff
@@ -77,6 +91,15 @@ public void scheduleButtonIsDisplayedProperly() {
                 .check(matches(allOf(isDisplayed(), hasChildCount(2), isClickable())));
     }
 
+    /**
+     * Clicking Schedule launches EventScheduleActivity.
+     */
+    @Test
+    public void roomInfoButtonLaunchesEventScheduleActivity() {
```
[the remark this answers is not in the document]
Comment (Rwothoromo) [reply app/src/androidTestMock/java/com/andela/mrm/room_availability/RoomAvailabilityActivityTest.java]: I'll update it right away.

File: app/src/androidTestMock/java/com/andela/mrm/room_availability/RoomAvailabilityActivityTest.java [hunk 3]
```diff
@@ -86,6 +109,15 @@ public void findRoomButtonDisplayed() {
                 .check(matches(allOf(isDisplayed(), hasChildCount(2), isClickable())));
     }
 
+    /**
+     * Clicking Find room launches FindRoomActivity.
+     */
+    @Test
+    public void roomInfoButtonLaunchesFindRoomActivity() {
```
[the remark this answers is not in the document]
Comment (Rwothoromo) [reply app/src/androidTestMock/java/com/andela/mrm/room_availability/RoomAvailabilityActivityTest.java]: I'll update it right away.

File: app/src/androidTestMock/java/com/andela/mrm/room_availability/RoomAvailabilityActivityTest.java [hunk 4]
```diff
@@ -86,6 +109,15 @@ public void findRoomButtonDisplayed() {
                 .check(matches(allOf(isDisplayed(), hasChildCount(2), isClickable())));
     }
 
+    /**
+     * Clicking Find room launches FindRoomActivity.
+     */
+    @Test
+    public void findRoomButtonLaunchesEventScheduleActivity() {
```
Comment (TheDancerCodes) [inline app/src/androidTestMock/java/com/andela/mrm/room_availability/RoomAvailabilityActivityTest.java]: Hi @Rwothoromo 
Looks like this one still needs an update. 
Kindly update to `findRoomButtonLaunches FindRoomActivity`

Comment (Rwothoromo) [reply app/src/androidTestMock/java/com/andela/mrm/room_availability/RoomAvailabilityActivityTest.java]: I have updated this as well.

**Comment by Rwothoromo** [comment]
@TheDancerCodes Coverage up by `2.0%` 🤓

**Comment by TheDancerCodes** [comment]
LGTM.
I am merging this now. @Rwothoromo

**Pull request merged**